lib/plugins/features/movies/index.js

```diff
@@ -30,6 +32,23 @@ exports.register = (server, options, next) => {
           query: MoviesListValidator
         }
       }
+    },
+    {
+      method: 'POST',
+      path: '/movies/{id}/locations',
+      config: {
+        handler: (request, reply) => {
+          reply(Controller.addLocationToMovie(request.params.id, request.payload.location_id));
+        },
+        validate: {
+          params: {
+            id: Joi.number().required()
+          },
+          payload: {
+            location_id: Joi.number().required()
```

Comment:
hm, i feel weird about this. Usually when we make POST requests of any sort, we try to pass in data containing the attributes and values we want to insert into the DB inside the payload; we usually do not specify the ID in the payload on a request like this.

For instance, I think what we conventionally do is make a `POST` request to `/movies/{id}/locations` then specify a payload of 
```
{
  name: 'San Francisco'
}

and in our controller cr?
```